src/components/v5/shared/Navigation/Sidebar/Sidebar.tsx

```diff
@@ -30,7 +30,7 @@ const Sidebar = React.forwardRef<HTMLDivElement, SidebarProps>(
         ref={ref}
         className={clsx(
           'top-[calc(var(--header-nav-section-height)+var(--top-content-height))]',
-          'z-sidebar hidden h-full w-fit flex-col items-start rounded-lg bg-gray-900 px-2 pb-4 pt-2 md:flex',
+          'z-sidebar hidden h-full w-fit flex-col items-start rounded-lg bg-gray-900 px-2 pb-[18px] pt-[14px] md:flex',
```

Comment:
Do we actually need to use pixels here? Can we sync with @melyndav on this?

For 14px, we could use pt-3.5, and for 18px, we can add a custom 4.5 value - https://tailwindcss.com/docs/customizing-spacing

Using specific pixel values feels less flexible to me